fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/ACTIVE/PERF.tsx

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+import { DropdownMenu } from 'instruments/src/MFD/pages/common/DropdownMenu';
+import { NumberInput } from 'instruments/src/MFD/pages/common/NumberInput';
+
+import { TopTabNavigator, TopTabNavigatorPage } from 'instruments/src/MFD/pages/common/TopTabNavigator';
+
+import { ArraySubject, DisplayComponent, FSComponent, Subject, Subscription, VNode } from '@microsoft/msfs-sdk';
+
+import { Button } from 'instruments/src/MFD/pages/common/Button';
+import { ActivePageTitleBar } from 'instruments/src/MFD/pages/common/ActivePageTitleBar';
+import { RadioButtonGroup } from 'instruments/src/MFD/pages/common/RadioButtonGroup';
+import { MfdComponentProps } from 'instruments/src/MFD/MFD';
+import { Footer } from 'instruments/src/MFD/pages/common/Footer';
+
+interface MfdFmsActivePerfProps extends MfdComponentProps {
+}
+
+export class MfdFmsActivePerf extends DisplayComponent<MfdFmsActivePerfProps> {
```

Comment:
**suggestion:** This page (and INIT) should probably not be under ACTIVE as the page is generic and can be used for both ACTIVE and SEC (all three secondary flight plans).

So I suggest we refactor this to instead be at the top of level of FMS and be instantiated for ACTIVE and SEC.

Reply:
I agree, will do.